src/components/data-management/LaunchAnalysisButton.jsx

```diff
@@ -99,29 +98,9 @@ const LaunchAnalysisButton = () => {
 
     const metadataKeysAvailable = activeExperiment.metadataKeys.length;
 
-    const allSampleFilesUploaded = (sample) => {
-      // Check if all files for a given tech has been uploaded
-      const { fileNames } = sample;
-      if (
-        !fileUploadSpecifications[sample.type].requiredFiles.every(
-          (file) => fileNames.includes(file.key),
-        )
-      ) { return false; }
-
-      let allUploaded = true;
-
-      // eslint-disable-next-line no-restricted-syntax
-      for (const fileName of fileNames) {
-        const checkedFile = sample.files[fileName];
-        allUploaded = allUploaded
-          && checkedFile.valid
-          && checkedFile.upload.status === UploadStatus.UPLOADED;
-
-        if (!allUploaded) break;
-      }
```

Comment:
valid is set to `true` on creation of the file in redux, so clearly it is an unnecessary check.

Reply:
I didn't understand the question. If you mean if there is a case in which it is set to false I don't think it does, it's just `true` and if it was false then it would never reach the point of `LaunchAnalysisButton` finding out it exists